docs/src/add-services/redis.md

```diff
@@ -189,62 +184,84 @@ const value = await client.get('x'); // returns 42
+Possible values are:
 
-* `noeviction`
-* `allkeys-lru`
-* `volatile-lru`
-* `allkeys-lfu` _(Available as of [Redis 4.0](https://redis.io/docs/reference/eviction/#the-new-lfu-mode))_
-* `volatile-lfu` _(Available as of [Redis 4.0](https://redis.io/docs/reference/eviction/#the-new-lfu-mode))_
-* `allkeys-random`
-* `volatile-random`
-* `volatile-ttl`
+| Value             | Policy description                                                                                          |
+|-------------------|-------------------------------------------------------------------------------------------------------------|
+| `allkeys-lru`     | Removes the oldest cache items first. This is the default policy.                                           |
+| `noeviction`      | New items aren’t saved when the memory limit is reached.                                                    |
+| `allkeys-lfu`     | Removes least frequently used cache items first.                                                            |
+| `volatile-lru`    | Removes least recently used cache items with the `expire` field set to `true`.                              |
+| `volatile-lfu`    | Removes least frequently used cache items with the `expire` field set to `true`.                            |
+| `allkeys-random`  | Randomly removes cache items to make room for new data.                                                     |
+| `volatile-random` | Randomly removes cache items with the `expire` field set to `true`.                                         |
+| `volatile-ttl`    | Removes cache items with the `expire` field set to `true` and the shortest remaining `time-to -live` value. |
 
-See the [Redis documentation](https://redis.io/docs/reference/eviction/) for a description of each option.
+For more information on the different policies,
+see the official [Redis documentation](https://redis.io/docs/reference/eviction/).
 
-## Using redis-cli to access your Redis service
+## Access your Redis service through the Redis CLI
 
-Assuming a Redis relationship named `applicationcache` defined in your app configuration:
+After you've [configured your Redis service](#usage-example),
+you can access it using the [Redis CLI](https://redis.io/docs/ui/cli/).
 
-{{< readFile file="src/registry/images/examples/full/redis.app.yaml" highlight="yaml" location=".platform.app.yaml" >}}
+Retrieve the hostname and port you can connect to 
+through the `PLATFORM_RELATIONSHIPS` [environment variable](../../development/variables/use-variables.md#use-platformsh-provided-variables).
+You can access the `PLATFORM_RELATIONSHIPS` environment variable directly [in your app](../../development/variables/use-variables.md#access-variables-in-your-app),
+for example using a Platform.sh configuration reader.
 
-and your service definition:
-
-{{< readFile file="src/registry/images/examples/full/redis.services.yaml" highlight="yaml" location=".platform/services.yaml" >}}
```

Comment:
:+1: But definitely good to get away from this confusing example that doesn't seem related to the command.